Assets/BossRoom/Scripts/Client/Game/Character/ClientCharacterVisualization.cs

```diff
@@ -301,12 +351,7 @@ void Update()
             if (m_ClientVisualsAnimator)
             {
                 // set Animator variables here
-                float visibleSpeed = 0;
-                if (m_NetState.NetworkLifeState.Value == LifeState.Alive)
-                {
-                    visibleSpeed = m_NetState.VisualMovementSpeed.Value;
-                }
-                m_ClientVisualsAnimator.SetFloat("Speed", visibleSpeed);
+                m_ClientVisualsAnimator.SetFloat("Speed", GetVisualMovementSpeed());
```

Comment:
Use m_SpeedVariableID here.

Reply:
also fixed some other non-hash-using lines in the same file

ok

Reply:
Awesome.